CONTRIBUTING.md

```diff
@@ -142,9 +142,11 @@ compound statement. One thing to watch out for is top-level sections in
 source files that are denoted by comments and are actually indented one
 level.
 
-Hoon will be a less familiar language to many contributors.  Until we publish
-an 'official' style guide, the best advice is again to mimic the code around
-you.  In general: the more recent the code, the more standard the style.
+Hoon will be a less familiar language to many contributors.  We've published
+some [style guidelines for Hoon][hoon], but above all you should try to mimic
+the style of the code around you.  With regards to the style used throughout
```

Comment:
Is this the advice we want to go with? So, if people are making changes/addition to zuse, and the code around it uses old syntax, they use old syntax too?